README.md

```diff
@@ -23,16 +23,61 @@ Several type of formulation are provided:
 
 The library is template-based, thus generic:  the curves can be of any dimension, and can be implemented in double or float and can work with kind variables like Vector, Transform, Matrix, ...
 
+## Installation
 
-Installation
--------------
+### Installation through robotpkg
+
+You can install this package through robotpkg. robotpkg is a package manager tailored for robotics softwares.
+It greatly simplifies the release of new versions along with the management of their dependencies.
+You just need to add the robotpkg apt repository to your sources.list and then use `sudo apt install robotpkg-py3\*-ndcurves`.
+
+If you have never added robotpkg as a softwares repository, please follow first the instructions from 1 to 3; otherwise, go directly to instruction 4.
+Those instructions are similar to the installation procedures presented in [http://robotpkg.openrobots.org/debian.html](http://robotpkg.openrobots.org/debian.html).
+
+1. Add robotpkg as source repository to apt:
+
+```bash
+sudo tee /etc/apt/sources.list.d/robotpkg.list <<EOF
+deb [arch=amd64] http://robotpkg.openrobots.org/wip/packages/debian/pub $(lsb_release -sc) robotpkg
+deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg
+EOF
+```
+
+2. Register the authentication certificate of robotpkg:
+
+```bash
+curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | sudo apt-key add -
+```
+
+3. You need to run at least once apt update to fetch the package descriptions:
+
+```bash
+sudo apt-get update
+```
+
+4. The installation of nd-curves:
+
+```bash
+sudo apt install robotpkg-py3\*-ndcurves # for Python 3
+
+sudo apt install robotpkg-py27-ndcurves # for Python 2
+```
+
+Finally you will need to configure your environment variables, e.g.:
+
+```bash
+export PATH=/opt/openrobots/bin:$PATH
+export PKG_CONFIG_PATH=/opt/openrobots/lib/pkgconfig:$PKG_CONFIG_PATH
+export LD_LIBRARY_PATH=/opt/openrobots/lib:$LD_LIBRARY_PATH
+export PYTHONPATH=/opt/openrobots/lib/python2.7/site-packages:$PYTHONPATH
```

Comment:
Instead of modifying the PYTHONPATH you can be python versions compatible by doing so: 
```bash
/opt/openrobots/lib/python2.7/site-packages > .local/lib/python2.7/site-packages/openrobots.pth
/opt/openrobots/lib/python3.8/site-packages > .local/lib/python3.8/site-packages/openrobots.pth
```

Reply:
This pth file are read upon python launch. So you can easily use several python installs by doing this.

These files needs to be where python look by default hence /home/user/lib/...

Reply:
It's fine, let's merge this!!!!